tests/api_test/api_test.cpp

```diff
@@ -1311,6 +1311,84 @@ TEST_CASE("test_ringbuffer_wraparound", "[stress]")
     // Unsubscribe from the ring buffer.
     ring_buffer__free(ring);
 
+    // Clean up.
+    bpf_object__close(object);
+}
+
+TEST_CASE("anusa_new_test", "[anusa_new_test]")
```

Comment:
what is the utility of this new test?

Reply:
This has been removed in the latest commit. I had added this to validate the regression tests.